Wiki: Add Advantages And Features Page (#254)
gridcoin-community/Gridcoin-Site: wiki/advantages-and-features.md

---
title: Gridcoin Advantages & Features
layout: wiki
---

# Competitive Advantages 

### General Advantages

> One of the main advantages to Gridcoin is that crunching ("mining")

Reviewer: ```suggestion
One of the main advantages of Gridcoin is that crunching ("mining")
```

Author: To clarify, both are grammatically correct. However using "of" sounds a bit more natural, so I implemented this

Reviewer: Thanks! Didn't know that.

does useful scientific research. Crunching helps science in areas like fighting 
diseases, finding asteroids, understanding subatomic particles, and more. In other
cryptocurrencies, mining doesn't produce any meaningful results. This means that 

> regardless of the number of Gridcoin you earn, crunching alone achieve something 

Reviewer: ```suggestion
regardless of the number of Gridcoin you earn, crunching alone achieves something 
```

Author: Oops not sure how I missed this one. Fixed that

things) see:
* [Comparison to CureCoin](#comparison-to-curecoin)
* [Comparison to OByte](#comparision-to-obyte)
* [Comparison to Primecoin](#comparison-to-primecoin)
* [Comparison to Tokens](#comparison-to-tokens)

Additionally, Gridcoin's system of scrapers offers a scalable system that 
minimizes that amount of centralization. Scrapers get BOINC statistics 
(which determine rewards) from the projects and then compress them down to only 
the useful stats --- reducing data sent and stored. The wallet compares the 
data from each scraper against each other limiting the trust needed. Scrapers are 
run by several trusted members of the community located around the world. 
Read more about scrapers on the [Scraper Wiki Page](Scraper "wikilink")

Further, Gridcoin has a voting system to help keep Gridcoin aligned with the
community and have meaningful discussions. Voting helps reduce conflicts by
not having unexpected changes and having a way to voice your opinion without 
needing to fork and split the community.

### Comparison to CureCoin
---
#### What's CureCoin?
CureCoin is a cryptocurrency that aims to reward folding@home work

> #### Why's Gridcoin Different?

Reviewer 2: I would prefer "Why is..."

Author: Changed it to `Why Gridcoin Is Different`. I think that might be a bit better

* CureCoin only rewards folding@home --- Gridcoin rewards many projects   
    * CureCoin is designed only for folding@home
        * Only one project can be rewarded
        * Expansion requires rework of the system

    * Gridcoin is designed for BOINC projects
        * Rewards many existing projects and allows for rewarding newly created BOINC projects
        * More rewarded areas of study and more scientists running and starting projects. 

^
^

* CureCoin involves Proof of Work ---  Gridcoin uses Proof of Stake
    * More energy used in Proof of Work
        * Less used energy on scientific work in CureCoin
    
    * With Gridcoin more energy goes directly to science
        * Less energy used by Proof of Stake

^
^

* Gridcoin has more active development and community
    * Comparison of development activity in the last month:
        * [Gridcoin's here](https://github.com/gridcoin-community/Gridcoin-Research/pulse/monthly) 
        * [CureCoin's here](https://github.com/cygnusxi/CurecoinSource/pulse/monthly)
 

> ### Comparison to OByte 

Reviewer 2: Clicking the link to this header isn't working for some reason (the others are)

Author: Ah oops I accidentally put comparis***i***on instead of comparison there on that link

---
#### What's OByte?

> Obyte (formally byteball) is a cryptocurrency that focuses on various different 

Reviewer 2: Capitalization of ByteBall

aspects such as smart contracts. Among other bots, it has a bot that distributes out 

> OBytes for work on [World Community Grid](https://www.worldcommunitygrid.org)

Reviewer 2: The correct plural here is "Bytes" or "Obyte bytes"

Author: Fixed that. I went with "Bytes"

> (note you can earn both Gridcoin and OBytes at the same time)

Reviewer 2: see above

#### Why's Gridcoin Different?

* OByte's main focus is different from Gridcoin 
    * Science aspect is a secondary or tertiary goal
        * Fewer OBytes sent to reward science and more goes to other distribution methods
    
    * For Gridcoin science is a primary goal
        * Larger percentage rewards science 

^
^

* OByte's bots are heavily centralized 
    * Happening through one bot that could stop at any point
        * Funding to this bot comes entirely through a grant from a fund 
          controlled currently by 4 people

    * In Gridcoin, coins are distributed as new coins in the network not through a single fund or bot

^
^

* OByte only rewards World Community Grid and no other BOINC project
    * On any other project, you can't earn OBytes for it

>     * With Gridcoin there are 18 project you can contribute to (as of 2020-08-08)

Reviewer 2: This is fine as long as we make a note to update this whenever the whitelist is changed

Author: Hmm actually I am able to pull directly from the whitelist.yml file so it would only need to be updated in one place with  `{{ site.data.whitelist.projects.size }}`. I'm going to do a PR real quick to update the whitelist and then add that I think

^
^

### Comparison to Primecoin
---
#### What Is Primecoin?
Primecoin is a crytpocurrency that uses a proof of work algorithm to 
find [Cunningham chains](https://en.wikipedia.org/wiki/Cunningham_chain) of prime numbers.

#### Why Gridcoin Is Different

> * Primecoin can only find prime numbers and do no scientific computation 

Reviewer: ```suggestion
* Primecoin can only find prime numbers and no other scientific computations
```

Author: I think I'm going to leave "do" in there to help clarify, but I will add "other" in there. I think both "computation" or "computations" are correct, oddly enough. I don't really know why that is. English grammar is weird.

    * Limited by Proof of Work algorithm 
        * Any expansion of Primecoin to other work would need
          to be a Proof of Work algorithm which need abilities that many

>           scientific problems lack 

Reviewer 2: this sentence is very awkward

Author: Is this any better? `Any expansion of Primecoin to other types of computations would need to fit a Proof of Work algorithm. These algorithms need abilities that many scientific problems lack`

            * See the challenges with even using primes in [Primecoin's whitepaper](https://primecoin.io/bin/primecoin-paper.pdf)

    * With Gridcoin more can be rewarded such as fighting disease, finding 

>       asteroids, and much more

Reviewer 2: too many uses of "more"

Author: Is this less repetitive? `With Gridcoin more can be rewarded such as fighting disease, finding 
      asteroids, and many other types of science`

^
^

*  Gridcoin has more active development and community
    * Comparison of development activity in the last month:
        * [Gridcoin's here](https://github.com/gridcoin-community/Gridcoin-Research/pulse/monthly) 
        * [Primecoin's here](https://github.com/primecoin/primecoin/pulse/monthly)

### Comparison to Tokens
---
#### What Are These Tokens
There are a variety of tokens that have tried/are trying to reward scientific 
work. 

#### Why's Gridcoin Different
* Most of these tokens are heavily centralized 
    * One person/group decides how they should be distributed
        * Many of these tokens have suddenly stopped because the people behind 
          the projects decided to stop giving tokens out.

    * Gridcoin has systems in place to not give one person or group the 
      ability to control distribution. 
        * The [Scrapers](scraper "wikilink"), Proof of Stake, and more allow for 
          a much more decentralized system.

^
^

* New tokens lack an existing community
    * Have to spend even more time on marketing, outreach, getting more 
    developers, etc.

>         * Chicken and egg problem

Reviewer 2: not explained

Author: Is this better `Have to overcome the chicken and egg problem for adoption`?
